### PR TITLE
feat!: Use String feePayer instead of Keypair

### PR DESCRIPTION
**Pull request by ChikinDeveloper**
No need to know about the fee payer private key here

### CODE REVIEW

**Comment by ChikinDeveloper** [comment]
warning • analysis_options.yaml:1:10 • The include file 'package:mews_pedantic/analysis_options.yaml' in \analysis_options.yaml' can't be found when analyzing 'ext_solana_dart_v2'. • include_file_not_found

But didn't change this file :/

**Comment by antoineherzog** [comment]
@iharobCP can you check?

**Comment by iharobCP** [comment]
Hello @ChikinDeveloper. Please pull master onto your branch so that checks can pass again. Then we can merge this PR. Thanks.

File: packages/solana/lib/src/helpers.dart [hunk 1]
```diff
@@ -52,6 +52,16 @@ Future<String> findProgramAddress({
   throw const FormatException('cannot find program address with these seeds');
 }
 
+/// Create a program address for [programId] and [seeds]
+Future<String> createProgramAddress({
```
Comment (iharobCP) [inline packages/solana/lib/src/helpers.dart:56]: How is this different from `findProgramAddress()`?

**Review by iharobCP — CHANGES_REQUESTED** [review]

File: packages/solana/lib/src/helpers.dart (hunk 1 above)
Comment (Ltei) [reply packages/solana/lib/src/helpers.dart:56]: findProgramAddress will find a nonce and add it to the seeds. This method is needed when you already know the nonce and want to get a program address

Comment (ChikinDeveloper) [reply packages/solana/lib/src/helpers.dart:56]: findProgramAddress will find a nonce to append to the seeds. createProgramAddress is used when you already know the nonce.

**Comment by ChikinDeveloper** [comment]
> Hello @ChikinDeveloper. Please pull master onto your branch so that checks can pass again. Then we can merge this PR. Thanks.

Looks like it's done, thanks antoine

File: packages/solana/lib/src/helpers.dart (hunk 1 above)
Comment (iharobCP) [reply packages/solana/lib/src/helpers.dart:56]: I see. What is the use case for this? Can you perhaps remove this change from this PR and create a different PR? It is unrelated to the other change. We are very interested in merging this, so if you could that would be great. Thanks.

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**